cli_tools/gce_vm_image_import/main.go

```diff
@@ -346,6 +375,12 @@ func main() {
 	metadataGCEHolder := computeutils.MetadataGCE{}
 	storageClient, err := storageutils.NewStorageClient(
 		ctx, createStorageClient(ctx), logging.NewLogger("[image-import]"))
+	defer storageClient.Close()
```

Comment:
I think Andrew correctly pointed out storageClient might be nil if there was an error creating it. Make sure this is handles. I assume reverting to having this call after the error check should work (if there was an error probably no need to call close on storageClient)